### DISCUSSION TITLE
Change which e2sSample to load on next boot

### DISCUSSION THREAD

**Discussion by Smarnz** [bangcorrupt/hacktribe#95] [Ideas]
Thought it may be possible to choose onboard the device which e2ssample file to load on next boot. Would effectively increase the amount of samples you could hold on the device, provided you only need one "sample pack" for any given project. 

Just an idea, not sure how hard it would be to implement but if possible think it would be a great upgrade :)

**Comment by bangcorrupt** [comment]
I like this idea.  Setting the filename when exporting samples and patterns would also be cool.  If I can get #27 working at some point, I'll try to make it so we can import sample packs/instruments without overwriting the existing samples.  I'd like to make a project file that holds a full session, rather than separate files for everything.

**Reply by ptowns** [reply]
Wait, I'm confused here. The e2ssample.all file is the one that's loaded by default on power up, but then once booted go to data utility, import sample and load any .all file you want. I have gigs and gigs of sample on my SD card. Max size on a .all file is like 27mb so a 1gb card can hold soooo many sample sets. 
Pro tip, keep a structure. Like 500 to 650 = kicks on your .ALL files so you can swap sample sets and patterns still 'work'

**Reply by bangcorrupt** [reply]
When exporting all samples it always overwrites e2sSample.all, it would be good to specify a different filename. I'd also like to be able to import sample kits without re-writing the whole collection.  Specifying a start slot and then importing something like `kicks.kit` could make sample management simpler.

**Reply by ptowns** [reply]
Got it, I never had much problems here since I do all my sample management on the computer.
